src/components/DataGraph/util.js

```diff
@@ -35,13 +39,92 @@ var parseC3Data = function(data) {
                     }
                     yAxisCount++;
                 }
-            }              
+            }          
         }
         allModelsData['columns'].push(xSeries);
         allModelsData['columns'].push(dataSeries);          
     }
-
     return [allModelsData, axisInfo];
 }
 
-export default parseC3Data
\ No newline at end of file
+
+var parseTimeSeriesForC3 = function(graph_data) {
+
+    var model = graph_data['id'];
+    var yUnits = graph_data['units'];
+    var C3Data = {
+        columns:[], 
+        types: {
+            model: 'line', 
+            'Annual Average': 'step',
+            'Seasonal Average': 'step'
+        }, 
+        labels: {
+            format: {
+                'Seasonal Average': function (v, id, i, j){
+                    if (i == 0 || i == 11){ return "Winter" }
+                    if (i == 3) { return "Spring" }
+                    if (i == 6) { return "Summer" }
+                    if (i == 9) { return "Fall" }
+                }
+            }
+        },
+        axes: {model:'y'},
+    };
+
+    var axisInfo = { 
+        x: { type:'category', categories:[] },
+        y: { label: { 'text': yUnits, 'position':'outer-middle' }} 
+    };
+
+    var tooltipInfo = {
+        grouped: true,
+        format: {
+            value: function (value) { return value + ' ' + yUnits }
+        } 
+    };
+
+    var monthlySeries = [model];
+    var springSeries = [];
+    var summerSeries = [];
+    var fallSeries = [];
+    var winterSeries = [];
+    var seasonalLabel = ['Seasonal Average'];
+    var annualSeries = ['Annual Average'];
+
+    var idx = 0;
+    for (let key in graph_data['data']) {
+        var val = graph_data['data'][key].toFixed(PRECISION);
```

Comment:
What's the purpose of rounding here? For the labels? If so, we should just round when creating the labels and leave the data intact.